apps/admin/src/app/dashboard/images/image-list.tsx

```diff
@@ -9,7 +9,7 @@ export const ImageList = ({ images }: { images: ImageUpload[] }) => {
         <a href={image.url} key={image.id} target="_blank" rel="noreferrer">
           {/*  eslint-disable-next-line @next/next/no-img-element */}
           <img
-            alt={`${image.id}`}
+            alt={image.id}
```

Comment:
[`@typescript-eslint/no-unnecessary-template-expression`](https://typescript-eslint.io/rules/no-unnecessary-template-expression): there's no need to wrap `image.id` with a template expression.